samples/TagHelperSample.Web/Areas/Admin/Views/Admin/Index.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+*@
+@{
+}
+
+<a asp-action="Index" asp-controller="Home" asp-area="">Go home!</a>
```

Comment:
> Thoughts on this?

I believe the pseudo-name of the "no area" area is `null`. If `string.Empty` is also correct, we should be fine w/ what @javiercn has done. Otherwise something like `asp-use-no-area` (name alternative) should add `["area"] = null` to the route values.